### PR TITLE
Support multiple private keys, not just one

### LINKED ISSUE #52: Support multiple private keys
This will help us with doing key rotation.

### PR DESCRIPTION
**Pull request by aermakov-zalando**
Support multiple private keys. Environment variable renamed to `SSH_PRIVATE_KEYS`, so beware! Fix #52.

### CODE REVIEW

File: src/org/zalando/stups/even/ssh.clj [hunk 1]
```diff
@@ -23,27 +25,38 @@
...
-  [hostname command {{:keys [user private-key port agent-forwarding timeout]} :config}]
+  [hostname command {{:keys [user private-keys port agent-forwarding timeout]} :config}]
   (log/info "ssh" user "@" hostname command)
   (let [agent (ssh-agent {:use-system-ssh-agent false
                           :known-hosts-path     "/dev/null"})
-        private-key-path (get-private-key-path private-key)]
-    (add-identity agent {:private-key-path private-key-path})
+        private-key-paths (write-private-keys private-keys)]
+    (doseq [path private-key-paths]
+      (println "adding identity" path add-identity)
```
Comment (harti2006) [inline src/org/zalando/stups/even/ssh.clj]: isn't there proper logger, instead of `println`?

Comment (aermakov-zalando) [reply src/org/zalando/stups/even/ssh.clj]: Oops, sorry. :( I left some debugging output in.

Comment (aermakov-zalando) [reply src/org/zalando/stups/even/ssh.clj]: Fixed now.

File: src/org/zalando/stups/even/ssh.clj [hunk 2]
```diff
@@ -23,27 +26,39 @@
...
+  [hostname command {{:keys [user private-keys private-key-password port agent-forwarding timeout]} :config}]
   (log/info "ssh" user "@" hostname command)
   (let [agent (ssh-agent {:use-system-ssh-agent false
                           :known-hosts-path     "/dev/null"})
-        private-key-path (get-private-key-path private-key)]
-    (add-identity agent {:private-key-path private-key-path})
+        private-key-paths (write-private-keys private-keys)]
+    (doseq [path private-key-paths]
+      (add-identity agent {:private-key-path path
+                           :passphrase       (some->> private-key-password
```
Comment (harti2006) [inline src/org/zalando/stups/even/ssh.clj]: Shouldn't this one be a single arrow `some->`, because `.getBytes` takes the string as **first** argument?

**Comment by aermakov-zalando** [comment]
👍

**Comment by harti2006** [comment]
:+1:

**Comment by harti2006** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**